flowpipe/node.py

```diff
@@ -161,11 +161,35 @@ def evaluate(self):
         return outputs
 
     @abstractmethod
-    def compute(self, *args, **kwargs):
+    def compute(self, *args, **kwargs):  # pragma: no cover
         """Implement the data manipulation in the subclass.
 
         Return a dictionary with the outputs from this function.
         """
+        raise NotImplementedError("Compute must be overwritten")
+
+    def __rshift__(self, other):
+        """Syntactic sugar for connecting outputs of this node based on names.
+
+        If other is an _InPlug, connect the output with matching name.
+        If other is an INode, connect all outputs with matching names.
+        """
+        if isinstance(other, _InPlug):
+            try:
+                self.outputs[other.name].connect(other)
+            except KeyError:
+                raise KeyError("No output named {0}".format(other.name))
+        elif isinstance(other, INode):
+            no_connection = True
+            for key in self.outputs.keys():
```

Comment:
should we make subplugs possible as well?

Reply:
Done. In order to make the code more readable, the order of checks is now from larger to more fine-grained objects, i.e. INode first, then InputPlugs

Reply:
Note that the behavior with SubOutputPlugs can be a bit unintuitive, since they only start existing once they are referred to explicitly for the first time. So if you never referred to an OutputPlug before, this `connect` method will connect the parent node instead. It still can be useful, e.g. when connecting the same node to multiple downstream nodes with SubPlugs, but it's a peculiarity to point out. Where would you have that kind of thing pointed out?

Reply:
Best to put that info in the docstring I'd say